app/src/main/java/ch/epfl/sdp/appart/database/local/LocalDatabase.java

```diff
@@ -35,26 +36,26 @@
  * disk and we can read it, or the caller must set it. One thing that is not
  * handled for now is the privacy of the users, maybe we don't want to store
  * everything "on disk".
- *
+ * <p>
  * Here is the file structure of the favorites folder :
  * favorites
- *          /currentUser
- *          profile_picture.jpeg
- *          currentUser.data
- *                      users/
- *                      users/
- *                                  {$user_id}/
- *                                              profile_picture.jpeg
- *                                              user.data
- *                      ${card_id}/
- *                                  data.fav
- *                                  Photo${number}.jpeg
- *                                  Panorama${number}.jpeg
+ * /currentUser
+ * profile_picture.jpeg
+ * currentUser.data
+ * users/
+ * users/
+ * {$user_id}/
+ * profile_picture.jpeg
+ * user.data
+ * ${card_id}/
+ * data.fav
+ * Photo${number}.jpeg
+ * Panorama${number}.jpeg
  */
 // @formatter:on
```

Comment:
Consider going to Settings -> Editor -> Code style -> Formatter control and enabling the formatter markers.